mysql/position.go

```diff
@@ -10,6 +10,8 @@ import (
 type Position struct {
```

Comment:
this structure means binlog position which can start a position-based replication, we should not add other fields. Maybe you can add a new interface `EventHandlerV2` with `OnXID(nextPos mysql.Position, xidEvent *replication.XIDEvent) error`.

And please translate the PR description to English so I can refer non-Chinese reviewers